packages/lib-classifier/src/store/ResourceStore.spec.js

```diff
@@ -71,20 +70,20 @@ describe('Model > ResourceStore', function () {
     const resetStore = ResourceStore.create(resourcesStub)
     resetStore.reset()
     expect(resetStore.resources.size).to.equal(0)
-    expect(resetStore.active).to.be.undefined
+    expect(resetStore.active).to.be.undefined()
     expect(Object.keys(resourceStore.headers)).to.have.lengthOf(0)
   })
 
   it('should use an existing resources object when `setActive` is called', async function () {
     await resourceStore.setActive('456')
     expect(resourceStore.active).to.deep.equal(resourcesStub.resources['456'])
-    expect(clientStub.panoptes.get.notCalled).to.be.true
+    expect(clientStub.panoptes.get.notCalled).to.be.true()
```

Comment:
The sinon-chai readme suggests using the chainable `not` for negation, so this should change to `expect(clientStub.panoptes.get).to.have.not.been.called()` for consistency.